app/controllers/snapshot_importer_controller.rb

```diff
@@ -0,0 +1,25 @@
+class SnapshotImporterController < ApplicationController
+  include HTTParty
+
+  def establecer_origen
+  end
+
+  def pide_lista_snapshots
+    @dominio = params[:dominio]
+    url = "#{@dominio}/snapshots.json"
+    response = HTTParty.get(url)
+    @snapshots = response.body.split(',')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.